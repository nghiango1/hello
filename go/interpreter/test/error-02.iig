// This should throw ERROR "unknown operator: BOOLEAN + BOOLEAN"
if ( 0 > 1 ) {
    return 1;
} else {
    if ( 2 < 3 ) {
        true + false;
        return 3;
    }
    return 2;
}
