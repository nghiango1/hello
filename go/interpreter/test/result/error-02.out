ERROR: unknown operator: BOOLEAN + BOOLEAN
